skyportal/handlers/photometry.py

```diff
@@ -68,6 +71,20 @@ def post(self):
             DBSession().add(p)
             DBSession().flush()
             ids.append(p.id)
+        if 'thumbnails' in data:
```

Comment:
I think we need to check a bit more carefully here that only thumbnails of allowed types are provided.  Also check size of resulting images (we should only allow within a certain range).  Do not allow the user to specify output filenames under any circumstances!

We should also note somewhere that this attached thumbnails to photometry 0.

Reply:
We _do_ check the thumbnail type - see line 79 below.

Reply:
No, not the file type, I mean the 'NEW', 'SD1', etc. — the 'type' dictionary entry.

Reply:
Ah I see - that's automatically taken care of by sqlalchemy.  If we try to create a thumbnail with a type not listed in the class definition, it errors.

Reply:
Just pushed up changes to address your comments - wanna take another look?

Reply:
I guess those errors are fine, although they may not provide the user with very sensible error messages. Thoughts?